frappe/templates/pages/integrations/stripe_checkout.py

```diff
@@ -26,7 +26,7 @@ def get_context(context):
 
 		context['amount'] = fmt_money(amount=context['amount'], currency=context['currency'])
 
-		if frappe.db.get_value(context.reference_doctype, context.reference_docname, "is_a_subscription"):
+		if frappe.get_meta(context.reference_doctype).has_field('is_a_subscription') and frappe.db.get_value(context.reference_doctype, context.reference_docname, "is_a_subscription"):
```

Comment:
```suggestion
		if (frappe.get_meta(context.reference_doctype).has_field('is_a_subscription') 
			and frappe.db.get_value(context.reference_doctype, context.reference_docname, "is_a_subscription")):
```

or move it to a function since the same check is used twice...

Reply:
moved to a function